nntrainer/src/conv2d_layer.cpp

```diff
@@ -53,10 +48,10 @@ int Conv2DLayer::initialize(bool last) {
     } else {
       bias.setZero();
     }
-    Tensor delK(gradDim);
+    Tensor delK(dim);
     delK.setZero();
 
-    Tensor delBias(gradBiasDim);
+    Tensor delBias(1, 1);
```

Comment:
Lets not use `(1, 1)` but use the dimension of the bias. Its more readable.

Reply:
fixed. thanks.